jquery.flot.js

```diff
@@ -1322,6 +1326,19 @@ Licensed under the MIT license.
                 });
             }
 
+            //adjust axis and plotOffset according to grid.margins
+            if (options.grid.margin) {
+                for (a in plotOffset) {
+                    var margin = options.grid.margin || 0;
+                    plotOffset[a] += typeof margin === "number" ? margin : margin[a] || 0;
```

Comment:
Just to make sure this line is doing the expected thing, refrase it like this:
typeof margin === "number" ? margin : (margin[a] || 0);

:)